Periscope_Web_Client.user.js

```diff
@@ -296,6 +297,7 @@ var Notifications = {
     }
 };
 function switchSection(section, param, popstate) {
+    ScrollPositions[document.URL.split('/')[3]]=document.body.scrollTop
```

Comment:
`ScrollPositions[document.URL.split('/')[3]] = window.pageYOffset;`
is better, works on chromium 69